plugin/test.vim

```diff
@@ -27,6 +27,7 @@ call s:extend(g:test#runners, {
   \ 'Lua':        ['Busted'],
   \ 'PHP':        ['Codeception', 'PHPUnit', 'Behat', 'PHPSpec', 'Kahlan', 'Peridot'],
   \ 'Perl':       ['Prove'],
+  \ 'Racket':     ['Rackunit'],
```

Comment:
I would prefer if it was named `RackUnit`, that's what the command name would be if the user chooses to generate it.